Crimson Dance support & CombinedDPS on the left side

Pegoth opened the pull request:
Added support for Crimson Dance and made Combined DPS visible on the left side.

For builds that do multiple damage types (impale + bleed + normal dps) combined DPS is very useful, as it was already calculated i think its good to show it on the left side.

Crimson Dance is calculated by multipying bleeding damage by 8 (meaning we assume that 8 bleed can be applied at all times).

LocalIdentity commented:
For Crimson Dance I believe that it would be better to have a box on the config screen to input the number of bleeds on a target, as you won't always have 8x stacks on an enemy. By default though you can have it assume 8 stacks if there is no number in the box.

Pegoth commented:
> For Crimson Dance I believe that it would be better to have a box on the config screen to input the number of bleeds on a target, as you won't always have 8x stacks on an enemy. By default though you can have it assume 8 stacks if there is no number in the box.

Changed the system to be exactly like the impale one and changed Crimson Dance to be an override to 8 max bleed, which doesn't really matter right now as I don't think there is any other maximum bleed stack source, but might be introduced later by GGG (who knows).

ppoelzl commented on Modules/CalcOffence-3_0.lua:
```diff
@@ -1648,7 +1648,10 @@ function calcs.offence(env, actor, activeSkill)
 				local rateMod = calcLib.mod(skillModList, cfg, "BleedFaster")
-				output.BleedDPS = baseVal * effectMod * rateMod * effMult
+				local maxStacks = skillModList:Override(cfg, "BleedStacksMax") or skillModList:Sum("BASE", cfg, "BleedStacksMax")
+				local configStacks = enemyDB:Sum("BASE", nil, "Multiplier:BleedStacks")
```
I suggest renaming both variables `maxStacks` and `configStacks` to something that incorporates "bleed", as such generic names could lead to further changes needed once a new, stack-based game mechanic is introduced. Maybe change them to `bleedStacksMax` and `bleedStacksConfig`?

Pegoth replied:
I made it exactly like how impale is handled. Also they are in an if that is only for bleed calculation.
